cpp/arcticdb/entity/metrics.hpp

```diff
@@ -34,6 +35,41 @@ const std::string PROMETHEUS_ENV_LABEL = "env";
 const int SUMMARY_MAX_AGE = 30;
 const int SUMMARY_AGE_BUCKETS = 5;
 
+class PrometheusConfig {
+public:
+    enum Model {
+        NO_INIT = 0,
+        PUSH = 1,
+        WEB = 2
+    };
+    PrometheusConfig() : model(Model::NO_INIT) {}
+
+    PrometheusConfig(const std::string& host,
+                     const std::string& port,
+                     const std::string& job_name, 
+                     const std::string& instance, 
+                     const std::string& prometheus_env, 
+                     const int model)
```

Comment:
Can't we bind the Model to a Python enum, and make Model an enum class, rather than passing as an int?